spec/features/admins/admin_dashboard_spec.rb

```diff
@@ -27,13 +27,14 @@
       expect(page).to have_button("Approve")
       expect(page).to have_content(@podcast.name)
       expect(page).to have_content(@podcast.description)
+      click_on("Approve")
     end
 
-    click_on("Approve")
-
     expect(page).to have_content("Application approved")
     expect(current_path).to eq('/dashboard')
-    
+
   end
 
+  #TODO make a sad path spec for a podcast that cant be approved. Later for EXTNS
```

Comment:
Making a sad path spec for a podcast that is not approved. Maybe we can make a Denied button? If so, what happens to the podcast ?!?!